cmd/config.go

```diff
@@ -42,9 +43,17 @@ type Config struct {
 	DisableEncoderMigrations  bool
 	HypersyncMaxQueueSize     uint32
 
+	// PoS Validator
+	PosValidatorSeed                       string
```

Comment:
Something I was telling Zordon is we should make it so these arguments accept EITHER a "seed hex" OR a "seed phrase." Not sure if you want to do that here, but I'm mentioning as it came in handy elsewhere. I think he made it so that if it starts with 0x, he reads it as a seed hex, otherwise he reads it as a seed phrase.

Reply:
I don't have a strong opinion on this. An optional check on the 0x prefix seems fine to add later on when needed. This uses the standard BIP39 seed phrase -> seed conversion, so it should just work out of the box to use a hex seed encoding